bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/productcarousel/ProductCarousel.java

```diff
@@ -27,7 +27,7 @@
 public interface ProductCarousel {
```

Comment:
One more thing. I'm thinking we should rename interface now as this now more than just a carousel thing. But I'm not really creative with a more common name.

Reply:
Well it's still a carousel, why change that? Only the source of the data changes ... but it's still  a carousel of products.

Reply:
Well was just an idea :-) lets keep it.